### PR TITLE
updates_content_styling

### PR DESCRIPTION
**Pull request by xritter1**
- Added templates for news, publications, events, people, tables, box
- Align styling (colors, font) as per Base4NFDI CI.

### CODE REVIEW

File: content/contact.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 # Contact
 
 ## Write to us!
-email@email.com
+You can contact the team via :envelope: email@email.com
```
Comment (dlehmenk) [inline content/contact.md]: i would use example.com, as email.com is a real domain

File: content/people.md [hunk 2]
```diff
@@ -2,7 +2,35 @@
 
 # People
 
-- Human 1
-- Human 2
+## Team as cards
+<div class="team-container">    <!-- Team as simple cards -->
+
+  <div class="team-member">
+    <img src="logo.png" alt="Team Member 1" class="team-photo">
```
Comment (dlehmenk) [inline content/people.md]: in my local test I don't see these pictures

**Review by dlehmenk — APPROVED** [review]

File: content/people.md (hunk 2 above)
Comment (xritter1) [reply content/people.md]: I couldn't find the right path again. I wonder if this is again a baseurl issue?

Comment (xritter1) [reply content/people.md]: I inserted real images now :)

**Pull request merged**